### ISSUE TITLE
focusing the table

### ISSUE THREAD

**Issue by bklingen** [jrowen/rhandsontable#144]
Is it possible to display the ``rhandsontable`` starting not in row 1, but in a scrolled-down state? The reason I'm asking is when I'm highlighting a specific row (supplied via the ``params`` option) in a larger table, the highlighted row may not be visible, and becomes only visible after scrolling. Is it possible to focus the table on that highlighted row? 
For instance, I have a table in a shiny app that looks like this (and, not visible, row 16 is highlighted):

![image](https://cloud.githubusercontent.com/assets/4784206/22085197/faa3f01e-dda0-11e6-95dc-04db3eea7b99.png)

But really I would like the ``rhandsontable`` to "automatically" scroll down so that the highlighted row is in focus:

![image](https://cloud.githubusercontent.com/assets/4784206/22085300/7b544524-dda1-11e6-9b30-a9cc12a12d8d.png)

In other words, is there a way to tell ``rhandsontable`` in which row it should start displaying the data?

**Comment by jrowen** [comment]
You could probably do this by adding JavaScript to your app (see [here](http://stackoverflow.com/questions/24053386/how-in-handsontable-scroll-to-row)).

**Comment by bklingen** [comment]
Great, thank you. 
As a feature request, if this is interesting to enough users, it may be nice to have a dedicated parameter ``scrolltorow=`` (or ``startwithrow=``) as part of the ``rhandsontable()`` or, maybe better, the ``hot_rows()`` call.

**Comment by bklingen** [comment]
handsontable has a ``selectCell`` command that, when set, would focus the table on the selected cell. This would be one way of focusing the table.
 
I went to the [handsontable tutorial](https://docs.handsontable.com/0.15.0-beta1/demo-highlighting-current.html) and modified the code shown there slightly, i.e.,

created a couple more artificial rows for the dataset
added: "height=100,"

and set "hot.selectCell(8,2)" in the last row:

```
 var
    data = [
      ['', 'Kia', 'Nissan', 'Toyota', 'Honda'],
      ['2013', 10, 11, 12, 13],
      ['2014', 20, 11, 14, 13],
      ['2015', 30, 15, 12, 13],
      ['2013', 10, 11, 12, 13],
      ['2014', 20, 11, 14, 13],
      ['2015', 30, 15, 12, 13],
      ['2013', 10, 11, 12, 13],
      ['2014', 20, 11, 14, 13],
      ['2015', 30, 15, 12, 13]
    ],
    container = document.getElementById('example1'),
    hot;
  
  hot = Handsontable(container, {
    data: data,
    minRows: 5,
    minCols: 6,
    height: 100,
    currentRowClassName: 'currentRow',
    currentColClassName: 'currentCol',
    rowHeaders: true,
    colHeaders: true
  });
  
  hot.selectCell(8,2);
```

With this, the handsontable (using JFiddler) correctly starts displaying the portion of the table in the scrolled down state, focusing (cell-highlight) on row 8.

Is there a way to use ``selectCell`` with rhandsontable? I tried 

```{r}
rhandsontable(data.frame(Col1=1:10,Col2=11:20), height=150) %>%
  hot_cols(renderer = "function(instance, td, row, col, prop, value, cellProperties) {
           Handsontable.TextCell.renderer.apply(this, arguments)
           td.selectCell=(8,1)
           }
           ")
```

hoping that this would focus the table on row 8 (or 7), but to no avail.
